Tests/RxSwiftTests/Atomic+Overrides.swift

```diff
@@ -1,5 +1,5 @@
 //
```

Comment:
You can delete this entire file and all depending on it.

Reply:
It's used in tests - should I just directly use the implementation ?